ci/run_truffle_tests.sh

```diff
@@ -49,6 +49,9 @@ run_tests $PROTOCOL_DIR/core
 cd $PROTOCOL_DIR/core
 
 $(npm bin)/truffle test $(find ../financial-templates-lib/test -name '*.js') --network ci
-$(npm bin)/truffle test $(find ../liquidator/test -name '*.js') --network ci
+$(npm bin)/truffle test $(find ../liquidator/test -name '*.js' ! -name 'OneInchExchange.js') --network ci
 $(npm bin)/truffle test $(find ../disputer/test -name '*.js') --network ci
-$(npm bin)/truffle test $(find ../monitors/test -name '*.js') --network ci
\ No newline at end of file
+$(npm bin)/truffle test $(find ../monitors/test -name '*.js') --network ci
+
+# Only run tests that needs the mainnet forking here
+$(npm bin)/truffle test ../liquidator/test/OneInchExchange.js
```

Comment:
Thoughts on just putting the tests that require a fork in a different directory, like `liquidator/fork-test`? That way, if we add more integration-y tests, we won't need to change this file for each one?